gateway/src/apicast/configuration_loader.lua

```diff
@@ -60,7 +60,7 @@ end
 _M.mock = mock_loader.save
 
 local function ttl()
-  return tonumber(env.get('APICAST_CONFIGURATION_CACHE'), 10)
+  return tonumber(env.get('APICAST_CONFIGURATION_CACHE') or 0, 10)
```

Comment:
`env.get` can return empty string, so it should do the `or` after `tonumber` or use `env.value` ?

Reply:
Right. Changed :+1: